src/test/regress/expected/multi_subquery.out

```diff
@@ -903,6 +903,28 @@ SELECT t1.event_type FROM events_table t1
 GROUP BY t1.event_type HAVING t1.event_type > avg(2 + (SELECT t2.value_2 FROM users_table t2 ORDER BY 1 DESC LIMIT 1))
 ORDER BY 1;
 ERROR:  cannot handle unplanned sub-select
+SELECT t1.event_type FROM events_table t1
+GROUP BY t1.event_type HAVING t1.event_type > avg(t1.value_2 + (SELECT t2.value_2 FROM users_table t2 ORDER BY 1 DESC LIMIT 1))
+ORDER BY 1;
+ERROR:  cannot handle unplanned sub-select
+RESET citus.coordinator_aggregation_strategy;
+SELECT t1.event_type FROM events_table t1
+GROUP BY t1.event_type HAVING t1.event_type > corr(t1.value_3, t1.value_2 + (SELECT t2.value_2 FROM users_table t2 ORDER BY 1 DESC LIMIT 1))
+ORDER BY 1;
+ERROR:  result "68_1" does not exist
```

Comment:
maybe open an issue for this to make sure we don't lose track. (or a PR)

Reply:
#3432

Reply:
#3432